pkg/restapi/oidc/operations.go

```diff
@@ -757,6 +783,31 @@ func updateEDVCapabilityInKeyStore(baseURL, keystoreID, controller, vaultID stri
 	return nil
 }
 
+func sign(r *http.Request, controller, compressedKMSCapability string, s signer) error {
+	action, err := operation.CapabilityInvocationAction(r)
+	if err != nil {
+		return fmt.Errorf("failed to determine webkms capability invocation action: %w", err)
+	}
+
+	r.Header.Set(
+		zcapld.CapabilityInvocationHTTPHeader,
+		fmt.Sprintf(`zcap capability="%s",action="%s"`, compressedKMSCapability, action),
+	)
+
+	hs := httpsignatures.NewHTTPSignatures(&zcapld.AriesDIDKeySecrets{})
+	hs.SetSignatureHashAlgorithm(&zcapld.AriesDIDKeySignatureHashAlgorithm{
+		KMS:    &zcapRemoteKMS{},
```

Comment:
if kms is not used here, then don't set KMS field. There's no need to create an empty zcapRemoteKMS implementation.

Reply:
@Baha-sk it is used: https://github.com/trustbloc/edge-agent/blob/10062645befa4e3857f53096d81e91c96cc162f4/pkg/restapi/oidc/webkms.go#L24-L26